web/templates/user_group_settings/browse_user_groups_list_item.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="group-row" data-group-id="{{id}}" data-group-name="{{name}}">
+<div class="group-row {{#if is_deactivated}}deactivated-{{/if}}user-group-info-box" data-group-id="{{id}}" data-group-name="{{name}}">
```

Comment:
I think would be good to just add a different class like `deactivated-group` for deactivated groups with `user-group-info-box` being present for all groups and then have the CSS selector accordingly.